src/talk/renderer/components/UserMenu.vue

```diff
@@ -18,54 +79,88 @@
 
 			<template #default>
 				<UiMenu aria-label="Settings menu">
-					<UiMenuItem tag="a" :href="userProfileLink" target="_blank">
-						<div><strong>{{ user['display-name'] }}</strong></div>
-						<div>{{ t('talk_desktop', 'View profile') }}</div>
-					</UiMenuItem>
-					<UiMenuItem v-if="userStatusStore.userStatus" tag="button" @click.native="isUserStatusDialogOpen = true">
-						<template #icon>
-							<NcUserStatusIcon :status="userStatusStore.userStatus.status" />
-						</template>
-						<template #default>
-							{{ visibleUserStatus }}
-						</template>
-					</UiMenuItem>
-					<UiMenuItem tag="button" @click.native="reload">
-						<template #icon>
-							<MdiReload />
-						</template>
-						{{ t('talk_desktop', 'Force reload') }}
-					</UiMenuItem>
-					<UiMenuItem tag="a" :href="$options.packageInfo.bugs" target="_blank">
-						<template #icon>
-							<MdiBug />
-						</template>
-						{{ t('talk_desktop', 'Report a bug') }}
-					</UiMenuItem>
-					<UiMenuItem tag="a" :href="talkWebLink" target="_blank">
-						<template #icon>
-							<MdiWeb />
+					<template v-if="userStatusSubMenuOpen">
+						<UiMenuItem tag="button" @click.native.stop="userStatusSubMenuOpen = false">
+							<template #icon>
+								<MdiChevronLeft :size="20" />
+							</template>
+							{{ t('talk_desktop', 'Back') }}
+						</UiMenuItem>
+						<UiMenuItem v-for="status in ['online', 'away', 'dnd', 'invisible']"
+							:key="status"
+							tag="button"
+							@click.native.stop="handleUserStatusChange(status)">
+							<template #icon>
+								<NcUserStatusIcon :status="status" />
+							</template>
+							{{ userStatusTranslations[status] }}
+							<template v-if="status === userStatus.status" #action-icon>
+								<MdiCheck :size="20" />
+							</template>
+						</UiMenuItem>
+					</template>
+
+					<template v-else>
+						<UiMenuItem tag="a"
+							:href="userProfileLink"
+							target="_blank">
+							<div class="user-menu__name">
+								<strong>{{ user['display-name'] }}</strong>
+								<em>{{ user.id }}</em>
```

Comment:
I'm afraid of possible overflow here (not for our instance, but others that use LDAP or another sources with long ids)
![image](https://github.com/nextcloud/talk-desktop/assets/93392545/2d03ada7-e9c0-46ad-9db9-801988c4a5e7)

Reply:
What about swap id <-> View profile?

![image](https://github.com/nextcloud/talk-desktop/assets/25978914/2e668b05-b0c5-4a4d-b2b6-e6cce80eb2b9)

Reply:
Or remove uid completely...

Reply:
Maybe `text-overflow: ellipsis;` after a certain width ?

Reply:
Nextcloud Desktop and Talk mobile client doesn't show email/uid anywhere at all.

But I personally would like to keep it.

@nickvergessen what do you think?

Reply:
I'm fine with a place swap only.
Ellipsizing might be tricky given that we don't know possible length of it for different auth methods, and also `display name` + `view profile` would differ from person and locale.

With possible multiple accounts feature (within one app) in future, it will be useful anyway